### ISSUE TITLE
Hardcoded configuration path breaks apps with different directory structure for app configuration

### ISSUE THREAD

**Issue by metaperl** [reahl/reahl#182]
On line 98 of [fixtures.py](https://github.com/reahl/reahl/blob/master/reahl-dev/reahl/dev/fixtures.py#L98) we see:

        config = StoredConfiguration('etc/')

and this is why I cannot successfully run pytest on my fixtures designed to create demo data.

Instead of `etc`, I have a directory named `stage` and under `stage` I have the directories:
1. 1-dev
1. 2-qa
1. 3-prod

I should be able to pass the configuration directory or set an environmental variable to set my configuration directory.

**Comment by IwanVosloo** [comment]
Yes, you're right. There are actually a number of things I'd like to parameterise for tests, such as the browser to use...

We will have to experiment regargind how to do this. Env vars is an option, but perhaps with [pytest commandline options](https://docs.pytest.org/en/latest/example/simple.html#dynamically-adding-command-line-options) we could do better.
